lawlibrary/views/legislation.py

```diff
@@ -10,27 +9,18 @@ def get_queryset(self):
         return super().get_queryset().filter(locality=None)
 
 
-class ProvincialLegislationView(TemplateView):
-    template_name = "lawlibrary/provincial_legislation.html"
+class LocalityLegislationView(BaseLocalityLegislationView):
     navbar_link = "legislation/provincial"
 
     def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
         codes = "mp ec nc kzn gp wc lp nw fs".split()
         provinces = Locality.objects.filter(code__in=codes)
-        groups = provinces[:5], provinces[5:]
-        return super().get_context_data(province_groups=groups, **kwargs)
+        context["province_groups"] = provinces[:5], provinces[5:]
+        context["page_heading"] = "Provincial Legislation"
+        return context
 
 
-class ProvincialLegislationListView(BaseLegislationListView):
-    template_name = "lawlibrary/provincial_legislation_list.html"
+class LocalityLegislationListView(BaseLocalityLegislationListView):
     navbar_link = "legislation/provincial"
```

Comment:
Move this `navbar_link` into the superclass (eg. as `legislation/locality`), and this class can fall away completely.